system/react/src/components/Banner.tsx

```diff
@@ -0,0 +1,125 @@
+import { Close, Product } from '@carbon/icons-react';
+import styled from '@emotion/styled';
+
+import { Button } from './Button';
+
+export const classySelector = '.banner';
+
+export const Wrapper = styled.div`
```

Comment:
These are the only imports/exports that should be in this file is what I meant.

```suggestion
import styled from '@emotion/styled';

export const classySelector = '.banner';

export const Banner = styled.div<{ 'data-variant'?: 'success' | 'info' | 'warning'; }>`
```

Reply:
Everything should be good now.